src/main/java/Services/CRUService.java

```diff
@@ -0,0 +1,11 @@
+package Services;
+
+import java.util.List;
+
+public interface CRUService<T> {
```

Comment:
I could not get what is the purpose of having 2 interfaces with identical internal structure. This interface is the same with DAOInterface.

Reply:
I agree. You have two identical interfaces.

Reply:
1 interface is related to entities and the other one is related to services. so entities has separate contracts and services has separate contracts. so can’t merge them into one